db/migrate/20150713125423_create_assessments.rb

```diff
@@ -0,0 +1,79 @@
+class CreateAssessments < ActiveRecord::Migration
+  def change
```

Comment:
Assignment Branch Condition size for change is too high. [45/15]<br>Method has too many lines. [60/10]